src/xcode/ENA/ENAUITests/HealthCertificate/ENAUITests_13_CreateHealthCertificate.swift

```diff
@@ -573,32 +608,135 @@ class ENAUITests_13_CreateHealthCertificate: CWATestCase {
 
 		snapshot("screenshot_2g_plus_certificate_overview")
 	}
+	
+	func test_screenshot_ExportCertificatesTooltip() throws {
```

Comment:
if we don't have screenshot, we can change the test name and move this up?

Reply:
thanks, done.